fuel-vm/src/interpreter/flow.rs

```diff
@@ -161,6 +161,13 @@ where
         let asset_id =
             AssetId::try_from(&self.memory[c as usize..cx as usize]).expect("Unreachable! Checked memory range");
 
+        let mut frame = self.call_frame(call, asset_id)?;
+
+        // FIXME: This is checking the cost after the db call has happened.
```

Comment:
Would be nice to reference this line in the issue just for better clarity

Reply:
https://github.com/FuelLabs/fuel-vm/pull/272#issuecomment-1376614156